File: examples/simple_examples/openai/sample.py
import os

from openai import OpenAI

client = OpenAI(
    base_url=f"https://platform.preferredai.jp/api/completion/v1",
    api_key=os.environ["PLAMO_API_KEY"],
    # other params...,
)

completion = client.chat.completions.create(
    model="plamo-2.0-prime",
    messages=[
        {"role": "system", "content": "あなたは学校の先生です"},
        {"role": "user", "content": "二次方程式の解の公式を端的に教えてください"},
    ],
)

print(completion.choices[0].message)

File: examples/simple_examples/openai-js/sample.js
const { OpenAI } = require('openai')

const openai = new OpenAI({
  baseURL: 'https://platform.preferredai.jp/api/completion/v1',
  apiKey: process.env.PLAMO_API_KEY
  // other params...
});

async function main() {
    const response = await openai.chat.completions.create({
    model: 'plamo-1.0-prime',
    messages: [
        {role: 'system', content:'あなたは学校の先生です'},
        { role: 'user', content: '二次方程式の解の公式を端的に教えてください' }
    ],
    });
    console.log(response.choices[0].message);
}

main();